pkg/server/sandbox_run.go

```diff
@@ -162,7 +161,15 @@ func (c *criContainerdService) RunPodSandbox(ctx context.Context, r *runtime.Run
 
 	container, err := c.client.NewContainer(ctx, id, opts...)
 	if err != nil {
-		return nil, fmt.Errorf("failed to create containerd container: %v", err)
+		if errdefs.IsNotFound(err) {
```

Comment:
A little bit concern about this. Is missing snapshot the only possible "NotFound" error?
Can we wrap `containerd.WithNewSnapshot` and do unpack their?

Reply:
Brilliant.